portal-frontend/src/app/features/notice-of-intents/edit-submission/proposal/roso/roso-proposal.component.html

```diff
@@ -128,6 +128,31 @@ <h2>Proposal</h2>
       <mat-hint class="subtext">Example: Aggregate, Extraction, Placer Mining, Peat Extraction, Soil etc.</mat-hint>
       <div class="subtext">Characters left: {{ 4000 - soilTypeRemovedText.textLength }}</div>
     </div>
+
+
+    <div class="form-row">
+      <div class="full-row">
+        <label for="projectDuration">Placement of Fill Project Duration</label>
```

Comment:
Should be 'Removal of Soil Project Duration', but perhaps also check with UX to see if we even need to explicitly add soil type to label here

Reply:
renamed